### ISSUE TITLE
*arr authentication no longer works.

### ISSUE THREAD

**Issue by InfinityDevTech** [goauthentik/authentik#9247]
**Describe the bug**
Hello! I have setup Radarr and Sonarr authentication using this [video](https://www.youtube.com/watch?v=RtPKMMKRT_E). This video is using the wiki page, I used the video as a sanity check just to make sure I wasnt crazy. I am using the Forward Auth proxy, and it is giving me an error in the Radarr console 

```
[Info] Radarr.Http.Authentication.BasicAuthenticationHandler: Basic was not authenticated. Failure message: Authorization header missing. 
[Info] Radarr.Http.Authentication.BasicAuthenticationHandler: AuthenticationScheme: Basic was challenged.
```

My config is perfect, NPM is setup correctly, it points correctly, but when I complete the authentik auth flow, that error pops up in the console, and it still prompts for the radarr password.

**To Reproduce**

1. Setup radarr basic HTTP auth.
2. Setup NPM and Authentik using wiki.
3. Attempt to login, with user permissions correct.
4. Cry.

**Expected behavior**
Authentik should be able to login with radarr correctly.

**Screenshots**
![image](https://github.com/goauthentik/authentik/assets/47366052/ded69ef7-cad6-4a63-a4cf-86c3ee5a0c3a)

**Logs**
```
{"event":"/outpost.goauthentik.io/auth/nginx","host":"radarr.infinity-dev.xyz","level":"info","logger":"authentik.outpost.proxyv2.application","method":"GET","name":"Radarr","remote":"MY PUBLIC IP ADDR","runtime":"1.157","scheme":"http","size":0,"status":200,"timestamp":"2024-04-13T20:46:13Z","user":"jwjoh","user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0"}
```
^^ This just gets logged whenever I attempt to login, this is the only thing lol

**Version and Deployment (please complete the following information):**

-   authentik version: 2024.2.2
-   Deployment: docker-compose

**Additional context**
No additional context

**Comment by InfinityDevTech** [comment]
Solved in #4381

**Comment by MikeWaggs** [comment]
> Solved in #4381

I'm running into that same error but only for radarr and prowlarr. Can you share which post in there fixed the issue for you?

**Comment by InfinityDevTech** [comment]
> > Solved in #4381
> 
> I'm running into that same error but only for radarr and prowlarr. Can you share which post in there fixed the issue for you?

Hey! 

My bad, it was [this post!](https://github.com/goauthentik/authentik/issues/4381#issuecomment-1605195711)

Have a good one!

**Comment by MikeWaggs** [comment]
> > > Solved in #4381
> > 
> > 
> > I'm running into that same error but only for radarr and prowlarr. Can you share which post in there fixed the issue for you?
> 
> Hey!
> 
> My bad, it was [this post!](https://github.com/goauthentik/authentik/issues/4381#issuecomment-1605195711)
> 
> Have a good one!

Perfect, fixed it for me too, not sure how I overlooked that! Thank you!
